errors.go

```diff
@@ -89,6 +89,10 @@ const (
 	ErrOffsetsLoadInProgress           KError = 14
 	ErrConsumerCoordinatorNotAvailable KError = 15
 	ErrNotCoordinatorForConsumer       KError = 16
+	ErrInvalidTopic                    KError = 17
+	ErrMessageSetSizeTooLarge          KError = 18
```

Comment:
You can't produce a message-set larger than your segment size. However the (default, rarely-changed) segment size is 1GB and the (default, rarely-changed) max request size is 100MB so this isn't ever a problem unless you've set up your cluster very oddly. I don't feel the need to add specific handling of this unless or until somebody complains that the configurable `MaxRequestSize` we already provide is not enough.